README.md

```diff
@@ -122,12 +122,44 @@ angular
   .controller('myController', MyController);
 ```
 
+### rollup-plugin-less-tilde-importer
+
+Provides ~ (tilde) prefix to tell less compiler that it should resolve imports using a configured array of paths.
+
+```js
+import configGenerator from '@ovh-ux/component-rollup-config';
```

Comment:
You can also add this line to your example I guess:

```js
import lessTildeImporter from '@ovh-ux/rollup-plugin-less-tilde-importer';
```

Reply:
nope, lessTildeImporter is just the option name added in pluginOpts, the rollup-plugin-less-tilde-importer is imported by component-rollup-config